### PR TITLE
Update info on compacting for tdb2 and add stats

### PR DESCRIPTION
**Pull request by OyvindLGjesdal**
Adding some info from the recent thread on jena-users discussed in [Large *.dat files in Fuseki](https://lists.apache.org/thread/r5xj4v8zsmy75gfrfs7vojhbx6r4kj9t)

### CODE REVIEW

File: source/documentation/tdb2/tdb2_admin.md [hunk 1]
```diff
@@ -39,7 +39,11 @@ latest view of the RDF dataset into that directory, then switch to using
 that generation of the database. 
 
 There is also a command line tool `tdb2.tdbcompact` to run the
-compaction process on a database not in use.
+compaction process on a database not in use. The command line
+option  `--deleteOld` removes the last database after compaction.
+
+Compaction can also be called from [the Fuseki HTTP Administration Protocol](/documentation/fuseki2/fuseki-server-protocol.html#compact)
+for live [Fuseki webapps](/documentation/fuseki2/fuseki-webapp.html).
```
Comment (OyvindLGjesdal) [inline source/documentation/tdb2/tdb2_admin.md:46]: I'm a bit unsure of what versions of fuseki which the administration protocol ships with. 

AFAIK this is all versions that comes with FUSEKI-UI, i.e all three versions bulleted in the fuseki-webapp link.

File: source/documentation/tdb2/tdb2_cmds.md [hunk 2]
```diff
@@ -105,3 +106,10 @@ The parallel loader runs all operations at once. It can deliver the best
 performance providing enough RAM is available and the persistent storage
 is SSD. It can consume all hardware resources, greatly impacting
 any other applications running.
+
+## `tdb2.tdbstats`
+
+Produce statistics for the dataset, which can be used for optimization rules. See the
+[TDB Optimizer description.](../tdb/optimizer.html#statistics-rule-file).
+
+For TDB2 the statistic files is read and placed in the `Data-NNNN` directory (`Data-0001/stats.opt`).
```
Comment (OyvindLGjesdal) [inline source/documentation/tdb2/tdb2_cmds.md:115]: Add info for location mentioned in https://lists.apache.org/thread/z4gogsnkhzfm1jwpqtpz3ht42qv8vz9q jena-dev thread. I also placed stats file in the parent dir of Data-NNNN before seeing it discussed.

File: source/documentation/tdb2/tdb2_admin.md (hunk 1 above)
Comment (afs) [reply source/documentation/tdb2/tdb2_admin.md:46]: Yes - it's part of fuseki-webapp.

File: source/documentation/tdb2/tdb2_cmds.md (hunk 2 above)
Comment (afs) [reply source/documentation/tdb2/tdb2_cmds.md:115]: I'm working on a compaction issue at the moment so I'll add a to-do to make sure the statis.opt is copied over.

**Comment by afs** [comment]
Stats files copied over during compaction: PR apache/jena#1452.

**Pull request merged**